doc/user/content/sql/functions/pushdown.md

```diff
@@ -11,17 +11,16 @@ menu:
 {{</ private-preview >}}
 
 `try_parse_monotonic_iso8601_timestamp` parses a subset of [ISO 8601]
-timestamps that is chosen to both:
-
-- Match the 24 character length output of the javascript [Date.toISOString()]
-  function and
-- For which the lexicographical order corresponds to chronological order.
-
-In contrast to other parsing functions, inputs that fail to parse return `NULL`
+timestamps that matches the 24 character length output
+of the javascript [Date.toISOString()] function.
```

Comment:
Should this link to some JavaScript docs?

Reply:
Plausibly! We're thinking about extending this function right now as well, though, in which case this would be not the right reference any longer... I'll follow up either way.